sentry-android-core/src/main/java/io/sentry/android/core/AppStartState.java

```diff
@@ -25,7 +25,7 @@
 
   private AppStartState() {}
 
-  static @NotNull AppStartState getInstance() {
+  public static @NotNull AppStartState getInstance() {
```

Comment:
`m`: Is this a breaking change?

Reply:
needed for react native, making it public isn't really a breaking change